scripts/verify-install.sh

```diff
@@ -20,18 +20,15 @@ MIN_REQUIRED_SIGNATURES=5
 # usernames in the list below are unique!
 KEYS=()
 KEYS+=("F4FC70F07310028424EFC20A8E4256593F177720 guggero")
-KEYS+=("15E7ECF257098A4EF91655EB4CA7FE54A6213C91 carlaKC")
 KEYS+=("E4D85299674B2D31FAA1892E372CBD7633C61696 roasbeef")
-KEYS+=("729E9D9D92C75A5FBFEEE057B5DD717BEF7CA5B1 wpaulino")
 KEYS+=("7E81EF6B9989A9CC93884803118759E83439A9B1 Crypt-iQ")
 KEYS+=("9FC6B0BFD597A94DBF09708280E5375C094198D8 bhandras")
-KEYS+=("E97A1AB6C77A1D2B72F50A6F90E00CCB1C74C611 arshbot")
-KEYS+=("EB13A98091E8D67CDD7FC5A7E9FE7FE00AD163A4 positiveblue")
```

Comment:
The removals should probably happen in a separate commit with a bit of a rationale.
Also, `positiveblue` signed `v0.16.0-beta` (and possibly later releases too), so we shouldn't yet remove them.